### DISCUSSION TITLE
Ortaklık

### DISCUSSION THREAD

**Discussion by Ege5560** [IBAX-io/go-ibax#1912] [Ideas]
### Clear and concise description of the problem

Değer kaybetmesini önlemek

### Suggested solution

Everyone knows that when the ibaxes obtained from the airdrop are exchanged, many people will sell them regardless of their value. Therefore, the value of ibaxc will decrease. The solution for this should be as follows. The people who hold ibaxc should be given a commission share from the people who sell it. An example person will sell 5 ibaxc, how many people have never sold, an amount such as 0.000001 will be shared. People who don't make any sales will not benefit from this commission. They have 500 ibaxc, for example, 200 sellers or even 1 seller will not be able to get it.

### Alternative
As an alternative, many people have already written a wallet application is a must. As an alternative to people who have sold Ibaxc but want to benefit from this commission, people who download the Wallet application to retain those people will receive 5 stars, and people who leave positive comments will receive a commission from these sellers.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Ege5560** [comment]
Yanlış bir şey yok isteyen sata bilir. Kilitli olsun demiyorum.Satmayan satandan komisyon alır. Satmayan kazanır tabi dilediği zamanda sata bilir.

**Reply by caner550** [reply]
Güzel olabilir . Ama ben ilk 1000 kişi unutur hiç gelmezse diye anladım .
Mantıklı dediğin 
15 bin kişi satar yarısı satmayan 1000 kişiye dağılır .
O bin kişi de hesabını unutmuşta olabilir .
Daha o zamana çok var . Yine de önlem ama çözüm değil
